static-test-tools/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG DOCKER_REGISTRY="docker.io/riot"
+arG DOCKER_REGISTRY="docker.io/riot"
```

Comment:
What? Why?

Reply:
No: How?

Honestly, no clue -- sorry it slipped in, thanks for spotting.

Reply:
(My best guess is that since my keyboard has ESC and ~ on the same key with different modifiers and ESC is something I use a lot, I hit ~ and doing so changed the case inadvertently.)